src/vm/memory/relocatable.zig

```diff
@@ -438,6 +610,42 @@ test "Relocatable: addMaybeRelocatableInplace should return an error if other is
     );
 }
 
+test "Relocatable: lt should return true if other relocatable is greater than or equal, false otherwise" {
+    try expect(Relocatable.new(-1, 2).lt(Relocatable.new(-1, 3)));
+    try expect(Relocatable.new(1, 5).lt(Relocatable.new(2, 4)));
+
+    try expect(!Relocatable.new(2, 4).lt(Relocatable.new(2, 4)));
+    try expect(!Relocatable.new(2, 5).lt(Relocatable.new(2, 4)));
```

Comment:
Can we add also a test for `Relocatable.new(2, 4).lt(Relocatable.new(2, 5));` I think it represents the second condition after the `or` inside your `lt` function?

Reply:
that is tested by 
>   Relocatable.new(-1, 2).lt(Relocatable.new(-1, 3))

Reply:
Yes ok, sounds good then